keyboards/boardwalk/boardwalk.h

```diff
@@ -0,0 +1,82 @@
+#ifndef BOARDWALK_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch? 

```suggestion
#pragma once
```

Reply:
Updated in https://github.com/qmk/qmk_firmware/pull/5195/commits/e23d9758c802d43be8763bd7982cfec5220f1d82